### DISCUSSION TITLE
Brim starting 90 degree offset from Seam setting

### DISCUSSION THREAD

**Discussion by StuweFPV** [supermerill/SuperSlicer#1713] [Q&A]
Can anybody help me with my issue please?
I set the seem to be in the back and use a brim. the brim ALWAYS starts 90 degree CCW from the Seam. Regardless of any seem setting or rotation of the object or disabling the prime. Even if i move the head with a custom G-Code right where the Brim should start, it jumps back to the 90 degree offset position and starts there.
I need as little movement as possible since i'm printing with LW-PLA and this is everytime causing a big blob on the start of the print. How can i control where the Brim starts - ideally where the print starts...
thank you for any help!

**Comment by supermerill** [comment]
I just look at it
There is an override for the skirt & brim to "nearest" (that was added in 2015, when there was only "nearest" and "aligned". The comment was `Force the 'nearest' strategy for starting skirt loops`)
And the gcode writer didn't parse the custom gcode to get the last position, so it thinks it starts at 0,0 and so you can't influence where the seam will be.

Obviously, you should want to have some sort of control on it.
I can try to parse the position from the last G1 in the start gcode.
I can also remove the override for the skirt if the option is 'rear' for the current config.
Please open an issue for that request.

**Comment by StuweFPV** [comment]
Excellent!!! thank you supermerill - i'll open a request for it. that would be a great help for my prints with LW-PLA
